draft-ietf-quic-recovery.md

```diff
@@ -615,6 +615,8 @@ are described in this section.
 bytes_in_flight:
 : The sum of the size in bytes of all sent packets that contain at least
   one retransmittable frame, and have not been acked or declared lost.
+  The size does not include IP or UDP overhead.  Ack only frames do not
+  count towards byte_in_flight.
```

Comment:
That is most of it, though if we wanted to include ack-only packets, it would be important to allow those to be sent immediately, even if congestion control indicated no data should be sent.

Martin, in regards to the pseudocode, I think it's simpler to think of not adding non-ACK data to packets if you're congested, rather than adding it and stripping it out.